lib/taskana-core/src/test/java/acceptance/task/UpdateManualPriorityAccTest.java

```diff
@@ -0,0 +1,207 @@
+import static acceptance.DefaultTestEntities.defaultTestWorkbasket;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import testapi.TaskanaInject;
+import testapi.TaskanaIntegrationTest;
+
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.api.models.Classification;
+import pro.taskana.classification.api.models.ClassificationSummary;
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.internal.jobs.JobRunner;
+import pro.taskana.common.test.security.WithAccessId;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.models.ObjectReference;
+import pro.taskana.task.api.models.Task;
+import pro.taskana.task.internal.builder.TaskBuilder;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+import pro.taskana.workbasket.internal.builder.WorkbasketAccessItemBuilder;
+
+@TaskanaIntegrationTest
+class UpdateManualPriorityAccTest {
+
+  @TaskanaInject TaskanaEngine taskanaEngine;
+  @TaskanaInject TaskService taskService;
+  @TaskanaInject ClassificationService classificationService;
+  @TaskanaInject WorkbasketService workbasketService;
+
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    defaultClassificationSummary =
+        defaultTestClassification().priority(9).buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary = defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+    defaultObjectReference = defaultTestObjectReference().build();
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_setPriorityToManualPriority_When_PositiveManualPriority() throws Exception {
+    Task task = createDefaultTask().manualPriority(123).buildAndStore(taskService);
+    Task result = taskService.getTask(task.getId());
+    assertThat(result.getPriority()).isEqualTo(result.getManualPriority()).isEqualTo(123);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_setPriorityAccordingToClassification_When_NegativeManualPriority() throws Exception {
+    Task task = createDefaultTask().manualPriority(-5).buildAndStore(taskService);
+    Task result = taskService.getTask(task.getId());
+    assertThat(result.getPriority()).isEqualTo(9);
```

Comment:
At first I was a little confused why the number `9` is expected here. After a few seconds I figured it out.
Suggestion: refactoring that number into a constant? When doing that we can give that number a meaning and it would look like this:
`assertThat(result.getPriority()).isEqualto(CLASSIFICATION_PRIORITY);`
which I think is a lot more readable.